src/main/java/com/datadoghq/sketch/ddsketch/encoding/VarEncodingHelper.java

```diff
@@ -0,0 +1,83 @@
+/* Unless explicitly stated otherwise all files in this repository are licensed under the Apache License 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2021 Datadog, Inc.
+ */
+
+package com.datadoghq.sketch.ddsketch.encoding;
+
+import java.io.IOException;
+
+public final class VarEncodingHelper {
+
+  private static final int MAX_VAR_LEN_64 = 9;
+  private static final int VAR_DOUBLE_ROTATE_DISTANCE = 6;
+
+  private VarEncodingHelper() {}
+
+  public static void encodeUnsignedVarLong(final Output output, long value) throws IOException {
+    for (int i = 0; i < MAX_VAR_LEN_64 - 1; i++) {
+      if (value >= 0 && value < 0x80L) {
+        break;
+      }
+      output.writeByte((byte) (value | 0x80L));
+      value >>>= 7;
+    }
+    output.writeByte((byte) value);
```

Comment:
Adapting the approach in `Serializer` might be helpful here. The varint of a 64 bit number `x` can be computed as `((63 - Long.numberOfLeadingZeros(x)`) / 7) + 1` which ranges from 1 to 10.

Reply:
Let's worry about this if it ever shows up in a profile...

Reply:
Actually, I pushed 7183fdac414c9901cf0514fbf979af95376f2ff8. I think it's fine either way.